src/vorta/assets/UI/mainwindow.ui

```diff
@@ -166,6 +166,11 @@
          <string>Archives</string>
         </attribute>
        </widget>
+       <widget class="QWidget" name="settingsTabSlot">
+        <attribute name="title">
+         <string>Settings</string>
```

Comment:
the tray bar entry is already called "Settings" (so all tabs are settings), so maybe rather call this one "Miscellaneous"?